R/helpers.R

```diff
@@ -822,13 +822,14 @@ cell_style_to_html.cell_text <- function(style) {
 cell_fill <- function(color = "#D3D3D3",
                       alpha = 1) {
```

Comment:
Consider making `alpha = NULL` the default, which will respect the `color`'s alpha (if present).

Reply:
This is a great suggestion. Thanks, I made the change and modified the surround documentation, and added some tests.